src/demo-app/list/list-demo.ts

```diff
@@ -69,4 +69,8 @@ export class ListDemo {
     this.selectedOptions = values;
     this.modelChangeEventCount++;
   }
+
+  alertItem(msg: string) {
+    alert(msg);
```

Comment:
Maybe use a snack bar instead?

Reply:
A snack bar will look nicer.  It can be done later in my opinion.  I am new to angular, so try to do small check in without taking too long to deal with merge conflict etc.

Reply:
This app is only for our own development, so it doesn't really matter what the action does.